packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,14 +1,239 @@
-# [HDS Components UI Kit](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=YeAupEyKdQqauO1k-1)
+# [HDS Components UI Kit v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=HYGTIoXBy2YkVWDP-1)
 
 ## December 20th, 2024
 
 `IconTile` - Removed the border and updated the colors for improved contrast and to create a distinctive look that aligns better with the surrounding elements.
 
-**Deprecated** HDS Components UI Kit v1.0. It will no longer receive updates or support. 
+`SideNav` - No longer deprecated due to adjustments in prioritization.
 
-Released HDS Components UI Kit v2.0.
+### Figma v2.0 Components
 
-`SideNav` - No longer deprecated due to adjustments in prioritization.
+Released HDS Components UI Kit v2.0. HDS Components UI Kit v1.0 is now deprecated and will no longer receive updates or support.
+
+!!! Insight
+
+**Migration Note**
+
+Many of the changes in the Figma v2.0 Library will result in a breaking change when replacing the v1.0 library components. For a comprehensive list of how these changes will impact your designs, refer to the [changelog document](https://docs.google.com/document/d/1q9IGtrsVOf3KrSBGb8AdF3OhPUm3fYAwPdhSml7onMM/edit?tab=t.0#heading=h.15f60fqiivvr).
+!!!
+
+#### General changes
+
+- Improved consistency in component naming conventions
+- Enhanced property panel customization options
+- Reduced variant complexity through boolean properties
+- Added local tokens for custom styling (hidden from publishing)
+- Standardized nested component structure
+
+#### Component organization updates
+
+- Form components now use "Form /" prefix instead of table of contents structure
+- Dialog-based components (Modal, Flyout) now use DialogPrimitive as base
+- Added [Template] components for common component combinations
+- Improved nested component exposure for easier customization
+- Standardized boolean property usage across components
+
+#### Component changes
+
+`Accordion` - Exposed nested instances for individual Accordion Items in the properties panel.
+
+`Alert`
+
+- Consolidated separate components (`page`, `inline`, and `compact`) into a single component.
+- Content properties (including icon swapping, text, and actions) have been exposed in the properties panel.
+    - To enable this, a nested local component has been created for alert content. This local component is shared with the Toast.
+
+`AppFooter`
+
+- Added local tokens for dark theme colors (these are hidden from publishing).
+- Removed `text` property on ".copyright" to avoid unnecessary editing.
+- In ".Link" subcomponent: added leading icon as a boolean property to match code.
+    - Leading icon now has instance swap.
+
+`ApplicationState` - Small cleanup changes only.
```

Comment:
Should this be included? I'm not sure what it means or if it is helpful for consumers to know.